src/server/src/models/user.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
```

Comment:
we store much more regarding a user - this is not part of this branch, just keep in mind

Reply:
What do you mean "not part of the branch"? Should I add more fields to the userSchema?